core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -743,7 +743,7 @@ class WorkspaceService(protected val userInfo: UserInfo,
           DBIO.from(Future(Map()))
         }
 
-        val query = for {
+        val query: DBIOAction[(Map[UUID, WorkspaceSubmissionStats], Seq[Workspace]), NoStream, Effect.Read with Effect.Write] = for {
```

Comment:
nit: there is a [type defined](https://github.com/broadinstitute/rawls/blob/develop/core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/package.scala#L10) that we can use in place of this long type annotation. Note that `ReadWriteAction` would actually be the equivalent substitution, but we aren't writing to the DB here as far as I know, so `ReadAction` should be sufficient.
```suggestion
        val query: ReadAction[(Map[UUID, WorkspaceSubmissionStats], Seq[Workspace])] = for {
```